pkg/cloudprovider/aws/apis/v1alpha1/provider_validation.go

```diff
@@ -22,6 +22,15 @@ import (
 	"knative.dev/pkg/apis"
 )
 
+const (
+	launchTemplatePath          = "launchTemplate"
+	securityGroupSelectorPath   = "securityGroupSelector"
+	fieldPathSubnetSelectorPath = "subnetSelector"
+	amiFamilyPath               = "amiFamily"
+	metadataOptionsPath         = "metadataOptions"
+	instanceProfilePath         = "instanceProfile"
```

Comment:
I'm not a fan of this pattern. If any of these values change, you'd need to change the variable name as well. The only benefit I can see compared to using the strings inline is compile time checks. I can see this happening due to an unsuccessful find+replace, but this feels like a stretch.

Reply:
I was more worried about typos in the hardcoded strings since they were hardcoded in multiple spots.

Reply:
I can live with it, as long as we don't move towards turning every string in the codebase into a constant.